src/controllers/connectionManager.ts

```diff
@@ -1634,67 +1649,12 @@ export default class ConnectionManager {
     }
 
     public async refreshAzureAccountToken(uri: string): Promise<void> {
-        const profile = this.getConnectionInfo(uri);
```

Comment:
most of this logic was just repeated from this.confirmEntraTokenValidity(connectionInfo.credentials);